src/test/java/io/debezium/connector/yugabytedb/YugabyteDBDatatypesTest.java

```diff
@@ -185,11 +205,15 @@ public void testVerifyValue() throws Exception {
         TestHelper.dropAllSchemas();
         TestHelper.executeDDL("postgres_create_tables.ddl");
         Thread.sleep(1000);
-        Configuration.Builder configBuilder = getConfigBuilder("public.t1");
+
+        String dbStreamId = TestHelper.getNewDbStreamId("yugabyte", "t1");
+        Configuration.Builder configBuilder = TestHelper.getConfigBuilder("public.t1", dbStreamId);
         start(YugabyteDBConnector.class, configBuilder.build());
         assertConnectorIsRunning();
         final long recordsCount = 1;
 
+        Thread.sleep(3000);
```

Comment:
why sleep? You have used sleep liberally, this is bad practice. Also, I don't understand, can you outline why we need it here? Is the database not ready?